pkg/render/config.go

```diff
@@ -29,6 +30,19 @@ type LibvirtConfig struct {
 	Replicas    string `json:"replicas"`
 }
 
+// OpenStackConfig contains specific config for OpenStack
+type OpenStackConfig struct {
+	ClusterName           string `json:"clusterName"`
+	ClusterID             string `json:"clusterID"`
+	Region                string `json:"region"`
+	AvailabilityZone      string `json:"availabilityZone"`
+	Image                 string `json:"image"`
+	ReleaseChannel        string `json:"releaseChannel"`
+	ContainerLinuxVersion string `json:"containerLinuxVersion"`
+	Replicas              string `json:"replicas"`
+	WithCreds             bool   `json:"withCreds"`
```

Comment:
WithCreds is used in aws so if set the actuator use those credentials to interact with the aws api, otherwise it uses the ami role in the given machine. How would this be handle for openstack?

Reply:
This would be handled by reading a `clouds.yaml` file from the system. It's a similar architecture to AWS, just different file format.